packages/fxa-settings/src/components/Settings/PageRecentActivity/en.ftl

```diff
@@ -1,13 +1,32 @@
-## Recent Activity
+## Recent account activity
+## All strings except title indicate an event that occurred from the user's account
+## These are displayed as a list with the date when the event occured
 
-recent-activity-title = Recent Account Activity
+recent-activity-title-v2 = Recent account activity
```

Comment:
This doesn't need a new ID since it's a capitalization change (other locales already style titles according to their own style guide).

That being said, we'll want to make sure the other en flavors (en-GB, en-CA) get updated.

Reply:
Gotcha! I'll revert to the previous ID in my next push.

Anything to do on my end to ensure the other EN varieties get updated?

Reply:
The easiest would be the manual method of making a suggestion with the capitalization changes for those locales in Pontoon after the source term gets merged.